src/main/resources/assets/lambdabettergrass/lang/ru_ru.json

```
@@ -0,0 +1,12 @@
+{
+  "lambdabettergrass.menu.info.1": "Мод использует ресурспак, чтобы определить, какие блоки затронуты.",
+  "lambdabettergrass.menu.info.2": "Включение или отключение функции улучшенного дёрна/снега на блоках",
+  "lambdabettergrass.menu.info.3": "нужно модифицировать с помощью ресурспака.",
+  "lambdabettergrass.menu.info.read_more": "Подробнее читайте на: %s",
+  "lambdabettergrass.menu.title": "Настройки LambdaBetterGrass",
+  "lambdabettergrass.menu.title.info": "Как работает мод?",
+  "lambdabettergrass.option.better_snow": "Улучшенный снег",
+  "lambdabettergrass.option.mode": "Улучшенный дёрн",
+  "lambdabettergrass.tooltip.better_snow": "Включает функцию улучшенного снега.\nЕсли этот параметр включен, в некоторых блоках может использоваться снежный вариант или будет добавлен слой снега, когда снег рядом.\nПримечание: это также работает для других подобных снегу блоков, у которых есть слои, если присутствуют правильные файлы.",
+  "lambdabettergrass.tooltip.mode": "Включает функцию улучшенного дёрна.\n - %s полностью отключает функцию улучшенного дёрна.\n - %s скопирует верхнюю текстуру со всех сторон блока дёрна. Похож на быстрый вариант OptiFine.\n - %s соединяет текстуры блоков дёрна, как детальный вариант OptiFine.\n - %s соединяет текстуры блоков дёрна и сгладит боковые текстуры ближайших блоков дёрна."
```

```suggestion
  "lambdabettergrass.tooltip.mode": "Включает функцию улучшенной травы.\n - %s полностью отключает функцию улучшенной травы.\n - %s копирует верхнюю текстуру на все стороны блока дёрна. Аналогично быстрому варианту OptiFine.\n - %s соединяет текстуры блоков дёрна аналогично настройке \"Детально\" в OptiFine.\n - %s соединяет текстуры блоков дёрна и сглаживает боковые текстуры ближайших блоков дёрна."
```
Снова предлагаю использовать "трава" вместо "дёрн", когда речь идёт о текстуре. +исправляю пару ошибок потери связи между словами.
Ах да, крайне рекомендую не использовать сочетание "как в Optifine". Всё-таки, LambdaBetterGrass – отдельный мод с улучшенными функциями, а не просто замена функций Optifine.